stac_fastapi/tests/resources/test_item.py

```diff
@@ -34,7 +34,7 @@ def rfc3339_str_to_datetime(s: str) -> datetime:
 
 
 @pytest.mark.asyncio
-async def test_create_and_delete_item(app_client, ctx, txn_client):
+async def test_create_item_indices(app_client, ctx, txn_client):
```

Comment:
Hi. Was changing the name of this test intentional?

Reply:
nice catch! It was a mistake...